### PR TITLE
FEATURE: Update api doc; WALLET-52

### PR DESCRIPTION
**Pull request by raulpaxful**

### CODE REVIEW

**Comment by sergeil** [comment]
@raulpaxful , let's have comments addressed in WALLET-52 and then merge this one.

File: docs/endpoints/offer.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Create an Offer
 | -------------          | ----------------------- | ----------------- | :------: |
 | offer_type_field       | String                  | 'buy' or 'sell' string |   Yes    |
 | currency               | String                  | 3 letter ISO code for fiat currency. 'USD' or any other. Case insensitive |   Yes    |
+| crypto_currency        | String                  | When not set, by default will be 'btc'. Crypto currency code: ‘btc', ‘usdt’ or 'eth’. |   No    |
```
Comment (GreatMoustaches) [inline docs/endpoints/offer.md:13]: Cryptocurrency is one word, not 2, remove space between them

File: docs/endpoints/trade.md [hunk 2]
```diff
@@ -40,10 +40,11 @@ Start a trade
 | Field name    |   Possible value  | Description   | Required |
 | ------------- | ----------------- | ------------- | :------: |
 | offer_hash    | String            | Hash ID of an offer | Yes |
-| satoshi    | Integer            | Trade amount in satoshis | read below |
-| fiat    | Float            | Trade amount in fiat currency | read below |
+| satoshi    | Integer            | Trade amount in satoshis | Read below |
```
Comment (GreatMoustaches) [inline docs/endpoints/trade.md:43]: Wrong description, correct one:
Trade amount in cryptocurrency.

File: docs/endpoints/trade.md [hunk 3]
```diff
@@ -40,10 +40,11 @@ Start a trade
...
+| crypto_amount | Integer | Trade amount in cyptocurrency. For btc trade in satoshi, for eth trade in gwei, for usdt trade in micro cents. 1 usdt = 1000000 micro cents. | Read below |
```
Comment (GreatMoustaches) [inline docs/endpoints/trade.md:44]: here is typo: Trade amoun in cRyptocurrency.

Comment (sergeil) [reply docs/endpoints/trade.md]: @raulpaxful , please fix the type and let's merge this PR.

**Review by GreatMoustaches — APPROVED** [review]

**Pull request merged**